### PR TITLE
Some minor style improvements and 2 test fixes

### PR DESCRIPTION
**Pull request by stefaang**
- fixed the 401 or 405 assertion always returning True
- tests were failing on a fresh install without test_user
- I made it a bit more pythonic.

### CODE REVIEW

**Comment by nicolaiarocci** [comment]
excellent. Tests are in dire need of refactoring (if not a complete rewrite).

**Comment by nicolaiarocci** [comment]
Thanks.

File: eve/methods/get.py [hunk 1]
```diff
@@ -184,7 +183,7 @@ def prune_aggregation_stage(d):
             abort(400, description="Aggregation query could not be parsed.")
 
         for key, value in query.items():
-            if key[0] != "$":
+            if key.startswith("$"):
```
Comment (mic-max) [inline eve/methods/get.py:186]: Was this change intended or should it be `if not key.startswith("$"):`

Comment (D-stefaang) [reply eve/methods/get.py:186]: Hmm not intended.

Comment (mic-max) [reply eve/methods/get.py:186]: I guess this code path isn't being tested?

Comment (nicolaiarocci) [reply eve/methods/get.py:186]: Thanks for the review @mic-max. Feel free to PR a fix.
